R/cal-apply.R

```diff
@@ -0,0 +1,98 @@
+#---------------------------------- Apply --------------------------------------
+
+#' Applies a calibration to a set of prediction probabilities
+#' @details It currently supports data.frames only. It extracts the `truth` and
+#' the estimate columns names, and levels, from the calibration object.
+#' @param .data An object that can process a calibration object.
+#' @param calibration The calibration object (`cal_object`).
+#' @param ... Optional arguments; currently unused.
+#' @examples
+#' w_calibration <- cal_estimate_logistic(segment_logistic, Class)
+#'
+#' cal_apply(segment_logistic, w_calibration)
+#' @export
+cal_apply <- function(.data, calibration, ...) {
```

Comment:
Sorry, but I dislike the argument name of `calibration`. Can we use something more general like `object`?

Reply:
I was gonna also change `.data` to `x`, but forgot before sending over the PR. Should we?